scripts/cleanup_edomain_templates.py

```diff
@@ -294,25 +294,31 @@ def run(**kwargs):
         if cli_provider and cli_provider != provider:
             continue
 
-        provider_mgmt = get_mgmt(provider)
+        print("\n--------Start of {}--------".format(provider))
```

Comment:
**OPTIONAL:** Make this message more specific to what its doing so something like:   Start of {} template cleanup

Reply:
The principle here is to give a reasonable amount of context so the reader will of the output will know what is going on at a high level.